frontend/appflowy_flutter/lib/mobile/presentation/database/card/card_detail/widgets/mobile_create_field_button.dart

```diff
@@ -40,7 +40,7 @@ class MobileRowDetailCreateFieldButton extends StatelessWidget {
           LocaleKeys.grid_field_newProperty.tr(),
           fontSize: 15,
         ),
-        onPressed: () => showCreateFieldBottomSheet(context, viewId),
+        onPressed: () => mobileCreateFieldWorkflow(context, viewId),
```

Comment:
I have no idea what to call this. I call it workflow because it first shows the field type bottom sheet. Upon selection, it then allows you to configure its type option. The field is only created when you click save. There is no bottom sheet called create field.

That being said, mobileCreateFieldWorkflow also sounds super weird. @LucasXu0 @Xazin